### DISCUSSION TITLE
How do I install macOS Monterey on Ubuntu

### DISCUSSION THREAD

**Discussion by petrasvestartas** [quickemu-project/quickemu#1084] [Quickemu]
#### Expected behaviour
Works

#### Actual behaviour

I installed quickmenu and gui and monterey like this:

```bash
sudo apt-add-repository ppa:flexiondotorg/quickemu
sudo apt update
sudo apt install quickemu

sudo add-apt-repository ppa:yannick-mauray/quickgui
sudo apt update
sudo apt install quickgui

quickget macos monterey
quickemu --vm macos-monterey.conf

```

https://github.com/quickemu-project/quickemu/assets/18013985/634a2ec5-0675-4ec1-9fd9-6de45db84343

![Screenshot_20240417_163623_Gallery](https://github.com/quickemu-project/quickemu/assets/18013985/5a8303bf-3890-41e9-ae89-cd4c54b748a1)

**Comment by TuxVinyards** [comment]
@flexiondotorg we so need a wiki for new users

@petrasvestartas please see the qqX FAQs > My VM won't work ....

**Comment by lj3954** [comment]
macOS Monterey is known to have many issues in quickemu. Catalina is the recommended release if the software you need is compatible with that release, and Ventura should be relatively stable as well if you need something newer.

**Comment by petrasvestartas** [comment]
@lj3954 Even for ventura, could are my steps for installation are correct?

I install ventura but it only does this and goes back to original window... please provide a dummy tutorial how to use it.

![Screenshot from 2024-04-18 13-32-04](https://github.com/quickemu-project/quickemu/assets/18013985/fb4ca2e3-d17e-46a4-bdec-c17f373d443b)c29a)
![Screenshot from 2024-04-18 13-32-29](https://github.com/quickemu-project/quickemu/assets/18013985/e359b962-e5ee-4c90-8b0d-4ea83aae9fcf)
![Screenshot from 2024-04-18 13-32-52](https://github.com/quickemu-project/quickemu/assets/18013985/497bf386-7fac-4933-a231-ef0178d76e0a)

**Comment by TuxVinyards** [comment]
It's all here https://github.com/TuxVinyards/qqX/wiki/FAQs-and-Help#macos-hangs-when-running-the-setup  

And the link at the bottom of that section #937

**Comment by petrasvestartas** [comment]
@TuxVinyards When he says: Using Quickemu from the master (what will become 4.9.3) I have successfully installed Ventura.

What does it actually mean?

**Reply by philclifford** [reply]
It means downloading (clone  (`git clone ...`) or Download zip ) the HEAD of the master branch and use `quickemu` and `quickget` from that unreleased position. The usual caveats of the "bleeding edge" apply, but there is a Makefile in the docs folder that can install the checked out/updated version.
